### PR TITLE
Adjusted magento versions link

### PR DESCRIPTION
**Pull request by lira**
## Description
Fixed Magento 2 link, set to Magento Marketplace releases

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

**Review by lamaralmp — APPROVED** [review]

File: guides/plugins/official/magento-two.es.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Para habilitar pagos con ticket > **Custom Checkout - Offline Payment Methods (T
 > Importante
 >
 > Antes de configurar el Pix como medio de pago, ten en cuenta:<br><br>
-> - [Descarga la última versión](https://br.wordpress.org/plugins/woocommerce-mercadopago/#description) para actualizar tu plugin de Mercado Pago.<br>
+> - [Descarga la última versión](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) para actualizar tu plugin de Mercado Pago.<br>
```
Comment (abderossi) [inline guides/plugins/official/magento-two.es.md:74]:
```suggestion
> - [Verifica la última versión](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) y actualiza tu plugin de Mercado Pago.<br>
```

File: guides/plugins/official/magento-two.en.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ To enable payments via ticket > C**ustom Checkout - Offline Payment Methods (Tic
 > Important
 >
 > Before setting up Pix as a payment method, remember to:<br><br>
-> - [Download the latest version](https://br.wordpress.org/plugins/woocommerce-mercadopago/#description) to update your Mercado Pago plugin.<br>
+> - [Download the latest version](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) to update your Mercado Pago plugin.<br>
```
Comment (abderossi) [inline guides/plugins/official/magento-two.en.md:94]:
```suggestion
> - [Verify the latest version](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) and update your Mercado Pago plugin.<br>
```

File: guides/plugins/official/magento-two.pt.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ Para habilitar pagamentos por boleto > **Custom Checkout - Offline Payment Metho
 > Importante
 >
 > Antes de configurar o Pix como meio de pagamento, lembre-se de:<br><br>
-> - [Baixar a última versão](https://br.wordpress.org/plugins/woocommerce-mercadopago/#description) para atualizar seu plugin do Mercado Pago.<br>
+> - [Baixar a última versão](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) para atualizar seu plugin do Mercado Pago.<br>
```
Comment (abderossi) [inline guides/plugins/official/magento-two.pt.md:95]:
```suggestion
> - [Verifique a última versão](https://marketplace.magento.com/mercadopago-core.html#product.info.details.release_notes) e atualize seu plugin do Mercado Pago.<br>
```

**Review by abderossi — CHANGES_REQUESTED** [review]
@pamalv7 sumé estos ajustes porque el link no lleva a descargar la última versión, sino a las release notes en donde se listan las últimas versiones. ¿Qué opinás?

**Review by abderossi — APPROVED** [review]

File: guides/plugins/official/magento-two.es.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ Para habilitar pagos con ticket > **Custom Checkout - Offline Payment Methods (T
 > Importante
 >
 > Antes de configurar el Pix como medio de pago, ten en cuenta:<br><br>
```
Comment (pamalv7) [inline guides/plugins/official/magento-two.es.md:73]:
```suggestion
> Antes de configurar Pix como medio de pago, ten en cuenta:<br><br>
```

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**